### PR TITLE
Feature admin delete components

### PR DESCRIPTION
**Pull request by asbamse**

### CODE REVIEW

File: src/app/admin/booth/booth-list/booth-list.component.ts [hunk 1]
```diff
@@ -10,15 +11,32 @@ import {BoothService} from "../../../shared/services/booth.service";
...
       this.booths = booths;
     }, error => {
+      this.errorOccured = true;
       this.errorMessage = error.error;
     });
   }
 
+  messageHandler(obs: Observable){
+    obs.subscribe(result => {
+      this.refresh();
```
Comment (JanvanZetten) [inline src/app/admin/booth/booth-list/booth-list.component.ts:44]: Should there be some positive message?

File: src/app/admin/user/user-list/user-list.component.ts [hunk 2]
```diff
@@ -10,15 +13,37 @@ import {UserService} from "../../../shared/services/user.service";
...
+      this.users = users;
+    }, error => {
+      this.errorOccured = true;
+      this.errorMessage = error.error
+    });
+  }
+
+  messageHandler(obs: Observable){
+    obs.subscribe(result => {
+      this.refresh();
```
Comment (JanvanZetten) [inline src/app/admin/user/user-list/user-list.component.ts:44]: again should there be a positive message?

**Review by Applemelon — DISMISSED** [review]
I love modals

**Review by Applemelon — APPROVED** [review]
I still love modals

**Pull request merged**